### PR TITLE
Extend OGGBundle with the DossierManager role.

### LINKED ISSUE #3672: Extend OGGBundle with additional property for `dossier_manager` permission
Similar we did for the excel import (see #3652) also the OGGBundle should be extended for repositoryfolders and dossiers.

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR extends the OGGBundler with the `DossierManager` role.

closes #3672

### CODE REVIEW

**Comment by elioschmutz** [comment]
@phgross I changed the mapping from required to optional (https://github.com/4teamwork/opengever.core/pull/3703/files#diff-90519493faa58540b0bd9761558f1ce3R63). 

Is there a reason, why it was required?

**Comment by phgross** [comment]
@elioschmutz not sure why the mapping was required, @lukasgraf do you remember?

**Comment by phgross** [comment]
@lukasgraf ?

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ Changelog
 2017.7.0 (unreleased)
 ---------------------
 
+- Extend OGGBundler with the DossierManager role. [elioschmutz]
```
Comment (lukasgraf) [inline docs/HISTORY.txt]: `OGGBundler` -> `OGGBundle`

**Comment by lukasgraf** [comment]
@elioschmutz @phgross no particular reason why `rolemap[role_shortname]` was used. It's BTW not about whether the mapping is optional or required, the behavior before was that *if* a `_permissions` mapping was provided, it needed to list every single permission (`add`, `edit`, ...). But that is unnecessary in my opinion, your change to `rolemap.get(role_shortname, [])` looks good 👍

**Comment by elioschmutz** [comment]
@lukasgraf 👍 updated PR

**Pull request merged**